docs/source/API/core/builtinreducers/MinLoc.rst

```diff
@@ -108,3 +108,64 @@ Additional Information
 * Requires: ``Index`` has ``operator =`` defined. ``Kokkos::reduction_identity<Index>::min()`` is a valid expression.
 
 * In order to use MinLoc with a custom type of either ``Scalar`` or ``Index``, a template specialization of ``Kokkos::reduction_identity<CustomType>`` must be defined. See `Built-In Reducers with Custom Scalar Types <../../../ProgrammingGuide/Custom-Reductions-Built-In-Reducers-with-Custom-Scalar-Types.html>`_ for details
+
+Example
+-------
+
+.. code-block:: cpp
+
+	#include <Kokkos_Core.hpp>
```

Comment:
Can you use for whitespaces instead of one tab?

Reply:
sure.  I'll have to do it by hand, though.

Reply:
Something like `%s/\t/    /g` in `vi`.